MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,750 @@
+
+{% tabs %}
+{% highlight c# %}
+
+    [DataFormDisplayOptions(RowOrder = 0, ItemsOrderInRow = 1)]
+    public string LastName { get; set; }
+
+    [DataFormDisplayOptions(RowOrder = 0, ItemsOrderInRow = 0)]
+    public string FirstName { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Using event
+
+The order of the fields can be changed by using the [ItemsOrderInRow] property in the `DataFormItem`.
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.ItemsOrderInRow = 1;
+            }
+            if (e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.ItemsOrderInRow = 0;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Grouping data fields
+
+It is possible to group some fields and set group name in the data form. You can expand or collapse the group by tapping the group item.
+
+Grouping can be achieved by defining [Display] attribute or by handling the `GenerateDataFormItem` event.
+
+### Using attributes
+
+{% tabs %}
+{% highlight c# %}
+
+public class ContactsInfo
+{
+    [Display(GroupName = "Name")]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name")]
+    public string MiddleName { get; set; }
+
+    [Display(GroupName = "Name")]
+    public string LastName { get; set; }    
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Using event
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName" || e.DataFormItem.FieldName == "MiddleName" || e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.GroupName = "Name";
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Changing order of the DataFormGroupItem
+
+The order of the `DataFormGroupItem` can be changed by using attributes. You can set the order of data form items in group by using the `Order` property along with `GroupName` property in display attribute.
+
+{% tabs %}
+{% highlight c# %}
+public class ContactInfo
+{
+    [Display(GroupName = "Name", Order = 0)]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name", Order = 1)]
+    public string LastName { get; set; }
+
+    [Display(GroupName = "Name", Order = 2)]
+    public string MiddleName { get; set; }
+}
+{% endhighlight %}
+{% endtabs %}
+
+The order of the `DataFormGroupItem` can also be changed by handling the `GenerateDataFormItem` event.
```

Comment:
The order of the editors in the group can also be changed by handling the  `GenerateDataFormItem` event.